<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConsulta</class>
 <widget class="QDialog" name="DialogConsulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Localidade:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateCheckin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLocalidade">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textResumo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinDiarias">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data de Checkin:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTipoQuarto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de quarto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de diárias:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConsultarDisponibilidade">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Verificar Disponibilidade</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
